src/sagemaker_tensorflow_container/training.py

```diff
@@ -125,7 +124,13 @@ def _wait_until_master_is_down(master):
             return
 
 
-def train(env):
+def _cmd_args(env, model_dir):
```

Comment:
I am not sure how much value this function provides. I would just fold it in _run_worker.

Reply:
it's used twice: once in `_run_worker` and once in `_train`